tests/countries/test_cape_verde.py

```diff
@@ -0,0 +1,109 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.countries import CapeVerde, CV, CAV
+from tests.common import CommonCountryTests
+
+
+class TestCapeVerde(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(CapeVerde, years=range(1973, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(CapeVerde, CV, CAV)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(CapeVerde(years=1972))
+
+    def test_2025_holidays(self):
+        self.assertHolidays(
+            CapeVerde(years=2025),
+            ("2025-01-01", "Ano Novo"),
+            ("2025-01-13", "Dia da Liberdade e Democracia"),
+            ("2025-01-20", "Dia da Nacionalidade e dos Heróis Nacionais"),
+            ("2025-04-18", "Sexta-feira Santa"),
+            ("2025-05-01", "Dia do Trabalhador"),
+            ("2025-06-01", "Dia Mundial da Criança"),
+            ("2025-07-05", "Dia da Independência Nacional"),
+            ("2025-08-15", "Dia da Assunção"),
+            ("2025-11-01", "Dia de Todos os Santos"),
+            ("2025-12-25", "Natal"),
+        )
+
+    def test_democracy_day(self):
+        name = "Dia da Liberdade e Democracia"
+        self.assertNoHolidayName(name, CapeVerde(years=1990))
+        self.assertHolidayName(
+            name,
+            (f"{year}-01-13" for year in range(1991, 2050)),
+        )
+        self.assertNoHolidayName(name, range(1980, 1990))
+
+    def test_no_democracy_day_before_1991(self):
+        self.assertNoHolidayName("Dia da Liberdade e Democracia", CapeVerde(years=1990))
```

Comment:
No need to separate them into two test cases.
```suggestion
    def test_democracy_day(self):
        name = "Dia da Liberdade e Democracia"
        self.assertHolidayName(name, (f"{year}-01-13" for year in range(1991, 2050)))
        self.assertNoHolidayName(name, CapeVerde(years=(1973, 1990)))
```